api/src/tests/unit/common/test_exception_handler_integration.py

```diff
@@ -0,0 +1,59 @@
+import json
+import re
+
+import pytest
+
+from common.exception_handlers import (
+    fall_back_exception_handler,
+    generic_exception_handler,
+)
+from common.exceptions import (
+    BadRequestException,
+    ExceptionSeverity,
+    MissingPrivilegeException,
+)
+
+
+def test_fall_back_exception_handler(caplog):
+    exception_response = fall_back_exception_handler("mock", exc=ZeroDivisionError())  # type: ignore
```

Comment:
Can make a simple mock request so as to avoid mypy ignore:
https://stackoverflow.com/a/67513232